docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '2'
+services:
+  database:
+    image: mongo:latest
+    container_name: database
+    environment:
+      - MONGO_DATA_DIR=/data/db
+      - MONGO_LOG_DIR=/dev/null
+    networks:
+      - default
+    volumes:
+      - ./data/db:/data/db
+    restart: always
+  api:
+    build:
+      context: .
+      dockerfile: ./docker/api
+    command: /app/bin/ssh_scan_api
+    ports:
+      - 8000:8000
+    environment:
+      - sshscan.api.bind=api
+      - sshscan.database.host=database
+      - sshscan.worker.token=d63920d7-82d1-4781-a6f9-369528c1f179
```

Comment:
FWIW, this is an example token for testing, this will be changed in deployment, perhaps being more upfront about this being an example would be worthwhile